Cutelyst/Plugins/Utils/Validator/validator.cpp

```diff
@@ -79,9 +82,8 @@ bool Validator::validate()
 
     bool valid = true;
 
-    for (int i = 0; i < d->validators.count(); ++i) {
-
-        ValidatorRule *v = d->validators.at(i);
+    for (std::vector<ValidatorRule*>::iterator it = d->validators.begin(); it != d->validators.end(); ++it) {
```

Comment:
I like long typing. :-P Mostly bad habit from having been too long in environments without C++11...